src/components/headers_overlay/headers_overlay.xml

```diff
@@ -101,25 +98,22 @@
         t-as="hiddenItem"
         t-key="hiddenItem_index">
         <div
-          class="o-unhide-buttons position-relative h-100 d-flex align-items-center"
+          class="o-unhide-buttons position-absolute h-100 d-flex align-items-center"
```

Comment:
You can change this line to `position-absolute h-100 d-flex align-items-center translate-middle-x gap-2`